telegram-client/helpers/parseLocation.ts

```diff
@@ -0,0 +1,12 @@
+const parseLocation = (text: string): string | null => {
+  const locationRegex = /#address [\'|\"](.*?)[\'|\"]/gm;
```

Comment:
Doesn't it make sense to move this `#address` hashtag to the list of them above?